### DISCUSSION TITLE
Contains in filter?

### DISCUSSION THREAD

**Discussion by Xyphyn** [pocketbase/pocketbase#1952] [Q&A] [answered]
I'm trying to filter my `posts` collection by all posts which community is in a user's followed communities. I've tried filtering by something like this:

`["pma8co4pbi6alsz", "v7ua5csufbl5h91"] ~ community.id`

What I expect this to do is check if that list contains the posts's community id. However, it doesn't return anything, despite there being posts with those communities. Am I doing this wrong? Where can I find the documentation for filters?

**Comment by ganigeorgiev** [comment, marked as answer]
Arrays literal are not supported as filter expression.

You can either try wrapping it inside single quotes:
```js
'["pma8co4pbi6alsz", "v7ua5csufbl5h91"]' ~ community.id
```

OR concatenate the expression with `||` (this is better since the id would be compared 1:1 and may have slightly better performance):

```js
// if community is a single relation "community" and "community.id" is the same
community = "pma8co4pbi6alsz" || community = "v7ua5csufbl5h91"
```

**Reply by Xyphyn** [reply]
Alright, thank you!

**Reply by ganigeorgiev** [reply]
> @Xyphyn  Where can I find the documentation for filters?

You can find some documentation about the filters in https://pocketbase.io/docs/api-rules-and-filters/#filters-syntax.
